ChangeLog.md

```diff
@@ -12,6 +12,7 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 - This ChangeLog!
 - Example code
 - Add better context to `DataApiException`s
+- Telemetry support
 
 ### Security fixes:
```

Comment:
This probably should have been mentioned in the previous PR, but would you mind fixing up this line to be `### Security` instead, to match how we've done it in other places ([and the examples](http://keepachangelog.com/en/1.0.0/))?